administrator/components/com_admin/sql/updates/postgresql/4.3.0-2023-03-14.sql

```diff
@@ -0,0 +1 @@
+UPDATE "#__extensions" SET "access" = 3 WHERE "name" = 'plg_system_guidedtours';
```

Comment:
```suggestion
UPDATE "#__extensions" SET "access" = 3 WHERE "type" = 'plugin' AND "element" = 'guidedtours' AND "folder" = 'system' AND "client_id" = 0;
```
I'd prefer the WHERE clause do use the combination of "type", "element", "folder" and "client_id" because we have a unique constraint for this combination, while for the "name" column we don't have a constraint to theoretically it might not be unique. But this is a suggestion only, I can live with it as it is since we have other update SQL scripts where we do it like this.